src/android/StatusBar.java

```diff
@@ -273,4 +285,13 @@ private void setStatusBarStyle(final String style) {
             }
         }
     }
+
+    private int getStatusBarHeight() {
+        Rect rectangle = new Rect();
+        Window window = cordova.getActivity().getWindow();
+        window.getDecorView().getWindowVisibleDisplayFrame(rectangle);
+        int statusBarHeight = Math.round(rectangle.top / Resources.getSystem().getDisplayMetrics().density);
```

Comment:
If the app is on the bottom of a split-screen, will this return the offset from the top of the screen? If so, this value will be incorrect.